tests/unit/test_service_steps.py

```diff
@@ -768,10 +770,54 @@ def test_eks_create_cluster_step_creation_sync():
 
 
 @patch.object(boto3.session.Session, 'region_name', 'us-east-1')
-def test_eks_delete_cluster_step_creation():
-    step = EksDeleteClusterStep("Delete Eks cluster", wait_for_completion=False, parameters={
-        'Name': 'MyCluster'
-    })
+def test_eks_create_cluster_step_creation_wait_for_completion():
+    step = EksCreateClusterStep("Create Eks cluster - WaitForCompletion",
+                                integration_pattern=IntegrationPattern.WaitForCompletion,
+                                parameters={
+                                    'Name': 'MyCluster',
+                                    'ResourcesVpcConfig': {
+                                        'SubnetIds': [
+                                            'subnet-00000000000000000',
+                                            'subnet-00000000000000001'
+                                        ]
+                                    },
+                                    'RoleArn': 'arn:aws:iam::123456789012:role/MyEKSClusterRole'
+                                })
+
+    assert step.to_dict() == {
+        'Type': 'Task',
+        'Resource': 'arn:aws:states:::eks:createCluster.sync',
+        'Parameters': {
+            'Name': 'MyCluster',
+            'ResourcesVpcConfig': {
+                'SubnetIds': [
+                    'subnet-00000000000000000',
+                    'subnet-00000000000000001'
+                ]
+            },
+            'RoleArn': 'arn:aws:iam::123456789012:role/MyEKSClusterRole'
+        },
+        'End': True
+    }
+
+
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_eks_create_cluster_step_creation_wait_for_task_token_raises_error():
+    error_message = re.escape(f"Integration Pattern ({IntegrationPattern.WaitForTaskToken.name}) is not supported for this step - "
```

Comment:
I was debating on adding a method in test/unit/utils to build the error message to avoid code repetition, but found that building the error message in each test improves readability. Thoughts?

Reply:
i like having it in the test for readability so don't have any objections to what you've gone with.